internal/api/threads.go

```
@@ -277,37 +277,79 @@ func (s *server) lockSession(ctx context.Context, tx pgx.Tx, id string) error {
+		folded, stop, err := events.PreviewTransition(ctx, tx, domain.ID(row.sessionID), events.ThreadTransition{
+			ThreadID: domain.ID(row.id), Status: domain.SessionTerminated})
+		if err != nil {
+			return row, err
+		}
+		after = stop
+		if folded == domain.SessionIdle && !sameStop(before, after) {
+			payload := map[string]any{}
+			if after != nil {
+				payload["stop_reason"] = after
+			}
+			raw, _ := json.Marshal(payload)
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `any` found

(errchkjson)